client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -206,9 +207,11 @@ class SiteImporterInputPane extends React.Component {
 		switch ( this.props.importerData.engine ) {
 			case 'wix':
 				return (
-					<div>
+					<div className="site-importer__site-importer-url-format">
```

Comment:
I would lean towards just totally removing this format info... I seriously question whether it is at all helpful to users.

Reply:
Good call, we'd talked about that during our initial audit but I forgot to follow through. Removed!

Reply:
Much better! And, it has the side-effect of bringing the "Continue" button closer to the URL field, which I think will help users out.